src/io/flutter/run/bazelTest/BazelTestConfigUtils.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.run.bazelTest;
+
+
+import com.google.common.annotations.VisibleForTesting;
+import com.intellij.icons.AllIcons;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.search.PsiElementProcessor;
+import com.intellij.psi.util.PsiTreeUtil;
+import com.jetbrains.lang.dart.psi.DartCallExpression;
+import com.jetbrains.lang.dart.psi.DartFile;
+import com.jetbrains.lang.dart.psi.DartStringLiteralExpression;
+import io.flutter.FlutterUtils;
+import io.flutter.dart.DartSyntax;
+import org.apache.commons.lang.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.List;
+
+public class BazelTestConfigUtils {
+
+  /**
+   * Widget test function as defined in package:flutter_test/src/widget_tester.dart.
+   */
+  public static final String WIDGET_TEST_FUNCTION = "testWidgets";
+
+  public static boolean isFlutterTest(DartFile file) {
+    return true;
```

Comment:
Should there be a TODO in this method???  If not, perhaps a comment as to why.

Reply:
This wasn't complete; I've gone back and finished this code.  PTAL.

For this section, it was definitely worth fixing instead of leaving it as a TODO.